packages/flutter_tools/test/integration.shard/test_test.dart

```diff
@@ -176,6 +226,22 @@ void main() {
     );
   });
 
+  testWithoutContext('flutter test should run all tests when tags are unspecified when --experimental-faster-testing is set', () async {
+    final ProcessResult result = await _runFlutterTest(
+      null,
+      automatedTestsDirectory,
+      flutterTestDirectory,
+      extraArguments: const <String>['--experimental-faster-testing'],
+    );
+    expect(
+      result,
+      ProcessResultMatcher(
+        exitCode: 1,
+        stdoutPattern: RegExp(r'\+\d+ -1: Some tests failed\.'),
+      ),
+    );
```

Comment:
This test currently isn't working because `flutter_tester` is returning exit code 0 even when printing `Some tests failed`.

Reply:
Some tests failed _should_ be coming from the script that's being run by flutter_tester right? How is it determining the exit code before this change?

Reply:
I've fixed this by updating the `main` method of `root_test_isolate_spawner.dart` to exit with the exit code set by `package:test`.